src/util/common.ts

```diff
@@ -113,9 +113,9 @@ export function fetchFromGitHub(repo: string, branch: string, path: string) {
 export function resolveMessage(ctx: ComponentContext, target: MessageOptions) {
   const { message: source } = ctx.data;
 
-  if (!target.content && source.content) target.content = '';
-  if (!target.embeds && source.embeds) target.embeds = [];
-  if (!target.components && source.components) target.components = [];
+  if (!('content' in target) && target.content !== null && source.content) target.content = '';
+  if (!('embeds' in target) && target.embeds !== null && source.embeds) target.embeds = [];
+  if (!('components' in target) && target.components !== null && source.components) target.components = [];
```

Comment:
```suggestion
  if (!('content' in target) && target.content !== null) target.content = '';
  if (!('embeds' in target) && target.embeds !== null) target.embeds = [];
  if (!('components' in target) && target.components !== null) target.components = [];
```
Checking `source` shouldn't be needed, but there is still the possibility of running into an error thrown by `slash-create` if none of them exist.